integration_test/ops_agent_test.go

```diff
@@ -27,12 +27,12 @@ PLATFORMS: a comma-separated list of distros to test, e.g. "centos-7,centos-8".
 The following variables are optional:
 
 REPO_SUFFIX: If provided, what package repo suffix to install the ops agent from.
+ARTIFACT_REGISTRY_REGION: If provided, signals to the install scripts that the
+    above REPO_SUFFIX is an artifact registry repo and specifies what region it
+	is in.
 AGENT_PACKAGES_IN_GCS: If provided, a URL for a directory in GCS containing
-.deb/.rpm/.goo files to install on the testing VMs.
-
-REPO_SUFFIX_PREVIOUS: Used only by TestUpgradeOpsAgent, this specifies which
```

Comment:
I like that we are getting rid of this `REPO_SUFFIX_PREVIOUS `, it simplifies things so much👍